pkg/kobject/kobject.go

```diff
@@ -83,6 +83,7 @@ type ServiceConfig struct {
 	Stdin         bool                `compose:"stdin_open" bundle:""`
 	Tty           bool                `compose:"tty" bundle:""`
 	MemLimit      yaml.MemStringorInt `compose:"mem_limit" bundle:""`
+	TmpFs         []string            `compose:"tmpfs" bundle:""`
```

Comment:
There may be some issues with this, see https://github.com/docker/libcompose/blob/f12d14623f73a0ca0d7fc37253a99165ae48d39f/config/types.go

It can either be a slice of a string, see: https://docs.docker.com/compose/compose-file/compose-file-v2/#tmpfs